### ISSUE TITLE
clarify versioning scheme

### ISSUE THREAD

**Issue by attila-lendvai** [romanz/trezor-agent#369]
i'm packaging `trezor-agent` and surrounding projects/dependencies like `libagent` for Guix.

this repo has multiple libs/apps in it, but obviously only one list of git tags.

the git tags seem to track `libagent`'s versioning, but there's a version in `agents/trezor/setup.py` that was not updated with git releases, even though its code is often changing (e.g. due to `libagent`'s API changing (?)).

the way i solve this in guix is that i append a `-1`, `-2`, `-n` revision postfix to the version of `trezor-agent` with each bump of the git tag.

isn't this an issue also in pipy? (i.e. the code changes but the version in `setup.py` remains the same)

so, to summarize: i can resolve this on the Guix side, but this may be an issue that deserves your attention. IOW, feel free to close this if it's a non-issue on your side.

**Comment by romanz** [comment]
First of all - I am very sorry for the very latest response!

My original idea was to version and release `libagent` and `trezor-agent` separately, but I think that you're right - and it would be easier to use a single version & tag for both of the projects.

**Comment by doolio** [comment]
@romanz have you considered creating separate repositories for hardware agent and `libagent`? They could be all be separate repositories under a Github Org. It would make packaging and versioning easier I imagine and avoid the confusion of all being under `trezor-agent`.

**Comment by attila-lendvai** [comment]
@doolio i think there's some architectural reason behind this. maybe `trezor-agent` is practically `libagent` plus some wrapping? but i may be way off here...

**Comment by attila-lendvai** [comment]
@romanz not sure whether you're still deciding, or it was just forgotten... so, it's a kind reminder that https://github.com/romanz/trezor-agent/blob/master/agents/trezor/setup.py still contains `version='0.12.0'` (and the same applies to the other agents).
